snyk/managers.py

```diff
@@ -138,31 +139,89 @@ def delete(self, key, value) -> bool:
 
 
 class ProjectManager(Manager):
-    def _query(self, tags: List[Dict[str, str]] = []):
+    def _rest_to_v1_response_format(self, project):
+        attributes = project.get("attributes", {})
+        settings = attributes.get("settings", {})
+        recurring_tests = settings.get("recurring_tests", {})
+        issue_counts = project.get("meta", {}).get("latest_issue_counts")
+
+        return {
+            "name": attributes.get("name"),
+            "id": project.get("id"),
+            "created": attributes.get("created"),
+            "origin": attributes.get("origin"),
+            "type": attributes.get("type"),
+            "readOnly": attributes.get("read_only"),
+            "testFrequency": recurring_tests.get("frequency"),
+            "isMonitored": True
+            if project.get("meta", {}).get("cli_monitored_at")
+            else False,
+            "issueCountsBySeverity": {
+                "low": issue_counts.get("low"),
+                "medium": issue_counts.get("medium"),
+                "high": issue_counts.get("high"),
+                "critical": issue_counts.get("critical"),
+            },
+            "targetReference": attributes.get("target_reference"),
+            "_tags": attributes.get("tags", []),
+            "browseUrl": f"https://app.snyk.io/org/{project.get('relationships', {}).get('organization', {}).get('data', {}).get('id')}/project/{project.get('id')}",
+            "importingUserId": project.get("relationships", {})
+            .get("importer", {})
+            .get("data", {})
+            .get("id"),
+            "owningUserId": project.get("relationships", {})
+            .get("owner", {})
+            .get("data", {})
+            .get("id"),
+        }
+
+    def _query(self, tags: List[Dict[str, str]] = [], next_url: str = None):
         projects = []
+        params = {}
         if self.instance:
-            path = "org/%s/projects" % self.instance.id
+            path = "/orgs/%s/projects" % self.instance.id if not next_url else next_url
+
+            # Append to params if we've got tags
             if tags:
                 for tag in tags:
                     if "key" not in tag or "value" not in tag or len(tag.keys()) != 2:
                         raise SnykError("Each tag must contain only a key and a value")
-                data = {"filters": {"tags": {"includes": tags}}}
-                resp = self.client.post(path, data)
-            else:
-                resp = self.client.get(path)
-            if "projects" in resp.json():
-                for project_data in resp.json()["projects"]:
+                data = [f'{d["key"]}:{d["value"]}' for d in tags]
+                params["tags"] = ",".join(data)
+
+            # Append the issue count param to the params if this is the first page
+            if not next_url:
+                params["meta.latest_issue_counts"] = "true"
```

Comment:
is this not needed on every request?

Reply:
The "next url" automatically includes this query param if it's passed to the first page.